source/SIL.AppBuilder.Portal.Frontend/src/ui/components/errors/parse-error.ts

```diff
@@ -6,6 +6,12 @@ export interface ParsedError {
 }
 
 export function parseError(error: any): ParsedError {
+  const errorDetail = error.data && error.data.errors && error.data.errors.length > 0 && error.data.errors[0].detail;
```

Comment:
This should be put after the instanceof checks.
otherwise you're totally ignoring the more specific error-handling.